recipes/openslide/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set OPENSLIDE_VERSION = "3.4.1" %}
+
+package:
+  name: openslide
+  version: {{OPENSLIDE_VERSION}}
+
+source:
+  url: https://github.com/openslide/openslide/releases/download/v{{OPENSLIDE_VERSION}}/openslide-{{OPENSLIDE_VERSION}}.tar.gz
+  md5: 88cfe1e9133e206c4e88402b7d19df31
```

Could you please change it to use ``sha256`` ?
```suggestion
  sha256: fed08fab8a9b1ded95a34e196652291127ebe392c11f9bc13d26e760295a102d
```